kofta/src/vscode-webview/components/BottomVoiceControl.tsx

```diff
@@ -54,7 +54,10 @@ export const BottomVoiceControl: React.FC<BottomVoiceControlProps> = ({
         key="leave-room"
         onClick={() => {
           wsend({ op: "leave_room", d: {} });
```

Comment:
this needs to be inside the if statement

Reply:
Fixed!